flags/14.0/vttestserver.txt

```diff
@@ -1,184 +1,145 @@
-      --num_shards string                                                     Comma separated shard count (one per keyspace) (default "2")
-      --onclose_timeout duration                                              wait no more than this for OnClose handlers before stopping (default 1ns)
-      --onterm_timeout duration                                               wait no more than this for OnTermSync handlers before stopping (default 10s)
-      --persistent_mode                                                       If this flag is set, the MySQL data directory is not cleaned up when LocalCluster.TearDown() is called. This is useful for running vttestserver as a database container in local developer environments. Note that db migration files (--schema_dir option) and seeding of random data (--initialize_with_random_data option) will only run during cluster startup if the data directory does not already exist. vschema migrations are run every time the cluster starts, since persistence for the topology server has not been implemented yet
-      --pid_file string                                                       If set, the process will write its pid to the named file, and delete it on graceful shutdown.
-      --planner-version string                                                Sets the default planner to use when the session has not changed it. Valid values are: V3, Gen4, Gen4Greedy and Gen4Fallback. Gen4Fallback tries the new gen4 planner and falls back to the V3 planner if the gen4 fails.
-      --planner_version string                                                planner_version is deprecated. Please use planner-version instead
-      --pool_hostname_resolve_interval duration                               if set force an update to all hostnames and reconnect if changed, defaults to 0 (disabled)
-      --port int                                                              Port to use for vtcombo. If this is 0, a random port will be chosen.
-      --pprof string                                                          enable profiling
-      --proto_topo string                                                     Define the fake cluster topology as a compact text format encoded vttest proto. See vttest.proto for more information.
-      --purge_logs_interval duration                                          how often try to remove old logs (default 1h0m0s)
-      --queryserver-config-transaction-timeout float                          query server transaction timeout (in seconds), a transaction will be killed if it takes longer than this value
-      --rdonly_count int                                                      Rdonly tablets per shard (default 1)
-      --remote_operation_timeout duration                                     time to wait for a remote operation (default 30s)
```

Comment:
Don't we need this for `vttestserver`. I see that `go/vt/topo/server.go:177` has 
```
FlagBinaries = []string{"vttablet", "vtctl", "vtctld", "vtcombo", "vtexplain", "vtgate",
		"vtgr", "vtorc", "vtbackup"}
```

Should this be in sync with (and getting replaced by) this in `go/vt/servenv/servenv.go:411`?
```
func RegisterFlagsForTopoBinaries(registerFlags func(fs *pflag.FlagSet)) {
	topoBinaries := []string{
		"vtbackup",
		"vtcombo",
		"vtctl",
		"vtctld",
		"vtgate",
		"vtgr",
		"vttablet",
		"vttestserver",
		"zk",
		"vtorc",
	}

	for _, cmd := range topoBinaries {
		OnParseFor(cmd, registerFlags)
	}
```

Reply:
vttestserver uses a different set of topo flags